src/huggingface_hub/hf_api.py

```diff
@@ -467,7 +467,7 @@ def erase_from_credential_store(username=None):
 class HfApi:
     def __init__(self, endpoint=None):
         self.endpoint = (
-            endpoint if endpoint is not None else os.getenv("HF_ENDPOINT", ENDPOINT)
+            endpoint if endpoint is not None else ENDPOINT
```

Comment:
I removed this, since `ENDPOINT` is already inheriting from the same env variable, to reduce the potential from divergence.